package.json

```diff
@@ -3,30 +3,15 @@
   "version": "2.4.0",
   "description": "Ambianic UI is the main user interface to Ambianic Edge deployments.",
   "author": "Ivelin Ivanov",
-  "license": "Apache-2.0",
-  "homepage": "https://ambianic.ai",
-  "keywords": [
-    "Home Automation",
-    "Ambient Intelligence",
-    "AI",
-    "Business Automation",
-    "Workspace Automation"
-  ],
-  "private": false,
-  "engines": {
-    "node": ">=12.10.0"
-  },
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "test": "npx cypress run",
     "codecov": "codecov -f coverage/coverage-final.json",
-    "e2e": "cypress open"
-  },
-  "funding": {
-    "type": "github",
-    "url": "https://github.com/sponsors/ambianic"
+    "e2e": "cypress open",
```

Comment:
Two comments here, for consistency, should we use the same naming convention for both types of tests:
`test:unit` and `test:e2e` instead of `test:jest` and `e2e`?